blockonomics-woocommerce.php

```diff
@@ -299,6 +299,10 @@ function show_basic() {
                         <tr valign="top">
                             <th scope="row"><?php echo __('Display Payment Page in Lite Mode (Enable this if you are having problems in rendering checkout page)', 'blockonomics-bitcoin-payments')?></th>
                             <td><input type="checkbox" name="blockonomics_lite" value="1" <?php checked("1", get_option('blockonomics_lite')); ?> /></td>
+                        </tr>
+                        <tr valign="top">
+                            <th scope="row"><?php echo __('No Javascript checkout page (Enable this if you have majority customer that use tor browser that don\t allow javascript)', 'blockonomics-bitcoin-payments')?></th>
+                            <td><input type="checkbox" name="blockonomics_nojs" value="1" <?php checked("1", get_option('blockonomics_nojs')); ?> /></td>
```

Comment:
Changed in text:

Enable this if you have majority customer that use tor like browser that block javascript